.bazelrc

```diff
@@ -28,6 +28,11 @@ build:devcontainer --disk_cache=/workspaces/magma/.bazel-cache
 common:devcontainer --repository_cache=/workspaces/magma/.bazel-cache-repo
 build:devcontainer --define=folly_so=1
 
+# TEST CONFIGS
+#  Bazel test runtime default: PATH=/bin:/usr/bin:/usr/local/bin
+#  Some python tests require access to /usr/sbin binaries (e.g. route, ifconfig)
+build --test_env=PATH=/bin:/usr/bin:/usr/local/bin:/usr/sbin
```

Comment:
No this is not possible here since $PATH would reference the PATH variable from the host system - and not the temporary PATH variable from the bazel runtime.